bindings/node/index.d.ts

```diff
@@ -493,6 +508,26 @@ export class ClientEncryption {
     callback: ClientEncryptionEncryptCallback
   ): void;
 
+  /**
+   * Encrypts a Match Expression or Aggregate Expression to query a range index.
+   *
+   * Only supported when queryType is "rangePreview" and algorithm is "RangePreview".
+   *
+   * @experimental The Range algorithm is experimental only. It is not intended for public use. It is subject to breaking changes.The aggregation or match expression you wish to encrypt.  The value must be in the form
+   *
+   * The expression to encrypt must be one of the following:
+   *  1. A Match Expression of this form:
+   *      `{$and: [{<field>: {$gt: <value1>}}, {<field>: {$lt: <value2> }}]}`
+   *  2. An Aggregate Expression of this form:
+   *      `{$and: [{$gt: [<fieldpath>, <value1>]}, {$lt: [<fieldpath>, <value2>]}]}`
+   *
+   *    `$gt` may also be `$gte`. `$lt` may also be `$lte`.
+   */
+  encryptExpression(
+    value: Document,
+    options: ClientEncryptionOptions
+  ): Promise<Document>
```

Comment:
Tiny stylistic nit, feel free to ignore:

```suggestion
  ): Promise<Document>;
```